client/scripts/CHAMPIONCAGE.as

```diff
@@ -480,6 +480,11 @@ package
          {
             var _loc2_:Object = _guardians["G" + param1];
             var _loc3_:Object = _loc2_.props;
+            for (var temp = 0; temp < _loc3_.health.length; temp++)
```

Comment:
Hey @AMXBus - would you mind explaining the reason for introducing this loop here? I noticed it's also called "temp", is this accidentally included in the PR?